Assignment2/marttamayy-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,32 @@
+{
+  "@context":{
+    "rdfs":"https://www.w3.org/2000/01/rdf-schema#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "subjs":"http://www.example2.es/assignment2/subjs#",
+    "fullName":"rdfs:label",
+    "hometown":"rdfs:label",
+    "age":"rdfs:label",
+    "alma":"rdfs:label",
+    "parentsNames":"rdfs:label",
+    "subjects":"subjs:subj",
+    "teachers":"subjs:teacher",
+    "description":"rdfs:comment"
+  },
+  
+    "fullName":"Marta Mayoral Jimenez",
+  	"hometown":"Alcorcon",
+  	"age":"22",
+  	"alma":"Universidad Politécnica de Madrid",
+ 	"parentsNames":"Isabel y Carlos",
+ 	"subjects":[
+      {"subj1":"Semantic Web, Linked Data and Knowledge Graphs",
+    	"teachers1":"Oscar Corcho",
+      	"desc1":"The objective of this course is to gain theoretical and practical knowledge about the methodological and technological foundations of the generation and publication of vocabularies and data on the Web of Linked Data, aswell as knowledge graphs"},

```

Comment:
Rename the subj1, teachers1, and desc1 properties as the names defined in the context. You don't need to define new ones, only reuse them for every subject.